src/genome_builds.php

```diff
@@ -244,7 +244,193 @@
+
+    if ($aAmountOfActiveGBs > 1) {
+        $aNavigation = array(
+            CURRENT_PATH . '?remove' => array('cross.png', 'Deactivate this genome build', 1),
+        );
+        lovd_showJGNavigation($aNavigation, 'genome_builds');
+    }
+
     $_T->printFooter();
     exit;
 }
+
+
+
+
+
+if (PATH_COUNT == 2 && ACTION == 'remove') {
+    // URL: /genome_builds/hg19?remove
+    // Deactivate one of the active genome builds.
+
+    define('PAGE_TITLE', lovd_getCurrentPageTitle());
+    define('LOG_EVENT', 'GenomeBuildRemove');
+
+    require ROOT_PATH . 'inc-lib-form.php';
+    require ROOT_PATH . 'class/object_genome_builds.php';
+
+
+    // Perform initial checks.
+    $sReason = '';
+    $aActiveBuilds = $_DB->query('SELECT id FROM ' . TABLE_GENOME_BUILDS)->fetchAllColumn();
+    $sID = lovd_getCurrentID();
+    $sColumnSuffix = $sID;
+
+    if (count($aActiveBuilds) < 2) {
+        // Check to make sure there would be a genome build left after the removal.
+        $sReason = 'there must be at least one reference genome left after the removal.';
+
+    } elseif (!in_array($sID, $aActiveBuilds)) {
+        // Check whether the given ID is one of the active IDs in the database.
+        $sReason = 'an invalid genome build was given.';
+
+    } else {
+        // Check to make sure that all variants are safely stored on the
+        //  genome builds that will remain active.
+        $sSQL = 'SELECT COUNT(id) FROM ' . TABLE_VARIANTS .
+            ' WHERE `VariantOnGenome/DNA/' . $sColumnSuffix . '` IS NOT NULL';
+
+        $aRemainingActiveBuilds = $aActiveBuilds;
+        if (($key = array_search($sID, $aRemainingActiveBuilds)) !== false) {
+            unset($aRemainingActiveBuilds[$key]);
+        }
+
+        foreach($aRemainingActiveBuilds as $aRemainingActiveBuild) {
+            $sSQL .= ' AND `VariantOnGenome/DNA/' . $aRemainingActiveBuild . '` IS NULL';
+        }
+
+        $bVariantsLostAfterRemovingGenomeBuild = $_DB->query($sSQL)->fetch()['COUNT(id)'] != 0;
+
+        if ($bVariantsLostAfterRemovingGenomeBuild) {
+            $sReason = 'not all variants that are mapped on this reference genome are safely mapped on a second build.';
+        }
+    }
+
+    // Throw error if any was found.
+    if ($sReason) {
+        lovd_showInfoTable('The genome build cannot be deactivated, since ' . $sReason, 'warning');
+        $_T->printFooter();
+        exit;
+    }
+
+
+    if (POST) {
+        lovd_errorClean();
+
+        // Check whether user has submitted and confirmed the form/action.
+        $bValidPassword = false;
+
+        if (empty($_POST['password'])) {
+            lovd_errorAdd('password', 'Please fill in the \'Enter your password for authorization\' field.');
+
+        } elseif (!lovd_verifyPassword($_POST['password'], $_AUTH['password'])) {
+            lovd_errorAdd('password', 'Please enter your correct password for authorization.');
+
+        } else {
+            $bValidPassword = true;
+        }
```

Comment:
Better stick to the LOVD defaults here;
- `lovd_errorClean()` cleans up any previous errors if there are any (almost never the case, but still).
- `lovd_errorAdd()` adds an error.
- `lovd_error()` then returns whether or not there are errors.
This way, you won't have to make your own variables.